lib/src/drag_marker_widget.dart

```diff
@@ -115,13 +117,7 @@ class DragMarkerWidgetState extends State<DragMarkerWidget> {
   void _pan(Offset localPosition) {
     final dragPos = _offsetToCrs(localPosition);
 
-    if (widget.mapState.isOutOfBounds(dragPos)) {
-      // cancels the dragging, needed when the app runs in a window and the
-      // cursor leaves the map while dragging. The on pan end event fails to
-      // fire when doing a quick movement
-      _end();
-      return;
-    }
+    // TODO: Please check if the bounds are correct or not before proceeding
```

Comment:
This pull request might introduce a bug on windowed devices.
The draggin might not stop when leaving the window while dragging a marker.

Reply:
Thanks @josxha
Completely forgot about that in the rush.

Can you or @ibrierley add that check in the new PR #39 ?